packages/ogmios/src/ogmiosToCore/tx.ts

```diff
@@ -249,15 +249,21 @@ const mapTxIn = (txIn: Schema.TxIn): Cardano.TxIn => ({
   txId: Cardano.TransactionId(txIn.txId)
 });
 
-const mapDatum = (datum: Schema.TxOut['datum']) => {
+const mapInlineDatum = (datum: Schema.TxOut['datum']) => {
+  if (!datum || typeof datum !== 'string') return;
```

Comment:
Can there be an empty datum string? Either way, I think `typeof` should be sufficient here.
```suggestion
  if (typeof datum !== 'string') return;
```